mmpose/datasets/builder.py

```diff
@@ -24,6 +26,38 @@
 PIPELINES = Registry('pipeline')
 
 
+def _concat_dataset(cfg, default_args=None):
```

Comment:
It seems an OpenMMLab convention that the ConcatDataset is implemented in `dataset/dataset_wrappers.py` and by inheriting the one from PyTorch and adding the subclass into the registry (For example in [mmcls](https://github.com/open-mmlab/mmclassification/blob/32d606e1a110188eb050e2d579120a3277b76957/mmcls/datasets/dataset_wrappers.py#L13) and [mmdet](https://github.com/open-mmlab/mmdetection/blob/ff9bc39913cb3ff5dde79d3933add7dc2561bab7/mmdet/datasets/dataset_wrappers.py#L17)). Can we also follow this practice?

Reply:
We do not need to rewrite ConcatDataset. The official one from PyTorch works fine.
Note: mmcls/mmdet also have _concat_dataset in builder.py